lib/cisco_node_utils/cmd_ref/vxlan_vtep_vni.yaml

```diff
@@ -16,7 +16,7 @@ all_vnis:
 ingress_replication:
   kind: string
   config_get_token_append: '/^ingress-replication protocol (\S+)$/'
-  config_set_append: '<state> ingress-replication protocol <protocol>'
+  config_set_append: '<state> ingress-replication protocol <protocol> ; end'
```

Comment:
To make sure that submode of ingress replication is not entered because there is no sub mode config being applied. This change may not be correcting anything but it keeps things clean.